lib/rubocop/cop/rails/rake_environment.rb

```diff
@@ -41,8 +41,26 @@ def on_block(node)
           end
         end
 
+        def autocorrect(node)
+          lambda do |corrector|
+            task = node.arguments[0]
+            corrected = corrected_task_name(task.source)
+
+            corrector.replace(task.loc.expression, corrected)
+            corrector.insert_after(node.loc.expression, ' :environment')
+          end
+        end
+
         private
 
+        def corrected_task_name(source)
+          if source.match?(/\w+:\w+(.*)?/)
```

Comment:
I think it's preferable not to use regular expression. I write a diff with several renames as below.

```diff
diff --git a/lib/rubocop/cop/rails/rake_environment.rb b/lib/rubocop/cop/rails/rake_environment.rb
index 5bf20c8e9..b08b40ac4 100644
--- a/lib/rubocop/cop/rails/rake_environment.rb
+++ b/lib/rubocop/cop/rails/rake_environment.rb
@@ -43,21 +43,20 @@ module RuboCop

         def autocorrect(node)
           lambda do |corrector|
-            task = node.arguments[0]
-            corrected = corrected_task_name(task.source)
+            task_name = node.arguments[0]
+            task_dependency = correct_task_dependency(task_name)

-            corrector.replace(task.loc.expression, corrected)
-            corrector.insert_after(node.loc.expression, ' :environment')
+            corrector.replace(task_name.loc.expression, task_dependency)
           end
         end

         private


-        def corrected_task_name(source)
-          if source.match?(/\w+:\w+(.*)?/)
-            source + ' =>'
+        def correct_task_dependency(task_name)
+          if task_name.sym_type?
+            task_name.source.delete(':|\'|"') + ': :environment'
           else
-            source.delete(':|\'|"') + ':'
+            "#{task_name.source} => :environment"
           end
         end

diff --git a/spec/rubocop/cop/rails/rake_environment_spec.rb b/spec/rubocop/cop/rails/rake_environment_spec.rb
index daf64e536..2beaf9dce 100644
--- a/spec/rubocop/cop/rails/rake_environment_spec.rb
+++ b/spec/rubocop/cop/rails/rake_environment_spec.rb
@@ -26,7 +26,7 @@ RSpec.describe RuboCop::Cop::Rails::RakeEnvironment do
     RUBY

     expect_correction(<<~RUBY)
-      task bar: :environment do
+      task 'bar' => :environment do
       end
     RUBY
   end
```